sass/routes/_settings.scss

```diff
@@ -271,9 +267,8 @@
 
                     .group--code {
                         select {
-                            @include appearance(none);
-
                             padding-right: 25px;
+                            -webkit-appearance: none;
```

Comment:
Non blocking: from what I can tell from caniuse, `appearance: none` is supported by all our supported browsers/versions meaning the `-webkit ...` version may not be needed anymore.